tests/inspector/cli-codegen-java.spec.ts

```diff
@@ -60,13 +60,13 @@ test('should print the correct context options when using a device', async ({ br
   expect(cli.text()).toContain(expectedResult);
 });
 
-test('should print the correct context options when using a device and additional options', async ({ browserName, runCLI }) => {
+test('should print the correct context options when using a device and additional options', async ({ browserName, runCLI, playwright }) => {
   test.skip(browserName !== 'webkit');
 
   const cli = runCLI(['--color-scheme=light', '--device=iPhone 11', '--target=java', emptyHTML]);
   const expectedResult = `BrowserContext context = browser.newContext(new Browser.NewContextOptions()
         .setColorScheme(ColorScheme.LIGHT)
-        .setUserAgent("Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Mobile/15E148 Safari/604.1")
+        .setUserAgent("${playwright.devices['iPhone 11'].userAgent}")
```

Comment:
This test is no longer testing anything!

Why does it change? Looks like a bug.